RUFAS/routines/field/crop/crop_management.py

```diff
@@ -49,19 +49,20 @@ def manage_harvest(self, harvest_op: HarvestOperation, field_name: str, field_si
             The object tracking the attributes of the soil profile.
 
         """
+        killed = False
```

Comment:
```suggestion
        is_killed = False
```
or just `is_killed_by_harvest`
Or `was_killed`/`was_killed_by_harvest`?

Reply:
Good suggestion